torch/csrc/jit/codegen/cuda/transform_replay.cpp

```diff
@@ -210,9 +212,34 @@ std::pair<TensorDomain*, unsigned int> TransformReplay::replayPasC(
     }
   }
 
-  // Map of consumer_CA_root_ids to related producer_CA_ids
   auto replay_root_map =
-      TensorDomain::mapRootCtoP(consumer, producer, consumer_CA_root_ids);
+      root_map.mapConsumerToProducer(consumer, producer, consumer_CA_root_ids);
+
+  // TODO: Remove checking with the old mapping
+  {
+    // Map of consumer_CA_root_ids to related producer_CA_ids
+    std::unordered_map<IterDomain*, IterDomain*> replay_root_map_old =
+        TensorDomain::mapRootCtoP(consumer, producer, consumer_CA_root_ids);
+    if (replay_root_map != replay_root_map_old) {
```

Comment:
is this temporary?

Reply:
Yes. The intention is to remove this block, however, this allows us to see where divergence happens between the current mapping with the new one. I'll remove it once we finalize the PR.

Reply:
PR looks good, I think we can remove these checks now.

Reply:
Removed.